graph_creator/views.py

```diff
@@ -42,70 +43,107 @@ def get(self, request, *args, **kwargs):
 class ReceiveData(View):
     """
     Receives and processes data from the remote edx-platform.
-    
+
     If the platform has already registered, a normal data exchange happens.
     Otherwise generates a secret token, (registers)saves it to DB with the edx-platform's incoming URL
-    and sends newly generated token to the edx-platform for further 
+    and sends newly generated token to the edx-platform for further
     data interchange abilities with the server.
     """
 
+    @staticmethod
+    def students_per_country(active_students_amount, students_per_country):
+        """
+        Method calculates amount of students per country.
+
+        Arguments:
+            active_students_amount (int): Count of active students.
+            students_per_country (list): List of dictionaries, where one of them is country-count accordance.
+                                         Amount of students without country is empty.
+
+        Returns:
+            students_per_country (list): List of dictionaries, where one of them is country-count accordance.
+                                         Amount of students without country has calculated.
+        """
+
+        students_per_country[0]['count'] = active_students_amount - sum(
+            [country['count'] for country in students_per_country]
+        )
+
+        return students_per_country
+
+    def create_instance_data(self, received_data, secret_token):
+        """
+        Method provides saving instance data as object in database.
+
+        Arguments:
+            received_data (QueryDict): Request data from edX instance.
+            secret_token (unicode): Secret key to allow edX instance send a data to server.
+                                    If token is empty, it will be generated with uuid.UUID in string format.
+        """
+
+        active_students_amount = int(received_data.get('active_students_amount'))
+        courses_amount = int(received_data.get('courses_amount'))
+        statistics_level = received_data.get('statistics_level')
+
+        instance_data = {
+            'active_students_amount': active_students_amount,
+            'courses_amount': courses_amount,
+            'secret_token': secret_token,
+            'statistics_level': statistics_level
+        }
+
+        if statistics_level == 'enthusiast':
```

Comment:
For now we exactly don`t sure about constants, because it is placed in edX`s overall settings.
We made dialog with @auraz about that, we decided leave it for current stage.